docs/standard/modernize-with-azure-and-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Choosing Azure compute platforms for container-based applications
+description: Modernize existing .NET applications with Azure Cloud and Windows containers | Choosing Azure compute platforms for container-based applications
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 05/04/2018
+---
+# Choosing Azure compute platforms for container-based applications
+
+As you have noticed after reading the previous sections, Azure is an open cloud that offers multiple choices. You can use the best fit for your needs, however, it also surfaces questions about what product/technology you should use for your containerized applications.
+
+As a *by-default* recommendation, the following is the main criteria recommended in this guidance:
+
+  - **Single monolithic app:** Choose Azure App Service
+  - **N-Tier app:** Choose orchestrators (AKS or SF) or App Service if you have a single or few backend service(s)
```

Comment:
need to introduce all these acronyms on first usage on this page

service(s) -> services

backend -> back-end